src/engine.rs

```diff
@@ -1532,6 +1564,8 @@ mod tests {
                 dir: dir.path().to_str().unwrap().to_owned(),
                 target_file_size: ReadableSize(1),
                 purge_threshold: ReadableSize(1),
+                format_version: Version::V1,
+                enable_log_recycle: false,
```

Comment:
Again, why change the config.

Reply:
Because the default setting of `Config` on version is `Version::V2`, not expected V1 in `cfg_v1`.

As the annotations said, `cfg_v1` should be initialized with `Verion::V1` and `cfg_v2` should be initialized with `Version::V2` for testing.